### DISCUSSION TITLE
Pybind11 for a already precompiled lib

### DISCUSSION THREAD

**Discussion by alexg93333** [pybind/pybind11#4618] [Q&A]
Is it possible to create pybindings for a already precompiled library if I have all the headers?

**Comment by lzqlzzq** [comment]
Yes, I think that can be done by utilizing [PCH (pre-compiled headers)](https://cmake.org/cmake/help/latest/command/target_precompile_headers.html) of cmake. And I wonder if the community would support PCH?
